README.md

```diff
@@ -81,10 +81,18 @@ openstack baremetal node manage $NODE_UUID --wait
 openstack baremetal node provide $NODE_UUID --wait
 ```
 
-And to deploy a node with the cirros image (image\_source above):
+And to deploy a master node with the coreos image (image\_source above) and passing in ignition config:
 
 ```
-openstack baremetal node deploy $NODE_UUID
+mkdir -p configdrive/openstack/latest
+cp ocp/master.ign configdrive/openstack/latest/user_data 
```

Comment:
```suggestion
curl -k https://ostest-api.test.metalkube.org:49500/config/master -o configdrive/openstack/latest/user_data 
```
This won't work without the network - `master.ign` is just a pointer to https://ostest-api.test.metalkube.org:49500/config/master. It should be fetched and placed there

Reply:
But we expect to have the network, so the wrapper ignition file should be fine, right? Or is the curl a workaround for the issues you're seeing in your environment?

Reply:
Neither helped in my case, so perhaps its fine to leave this as is